all_protos/injective/exchange/v2/events.proto

```
@@ -150,6 +159,13 @@ message EventSubaccountBalanceTransfer {
   cosmos.base.v1beta1.Coin amount = 3 [ (gogoproto.nullable) = false ];
 }
 
+message EventSubaccountRiskProfileUpdated {
+  string subaccount_id = 1;
+  SubaccountRiskProfile previous_profile = 2 [ (gogoproto.nullable) = false ];
```

Buf (1.70.0)

[error] cannot find `SubaccountRiskProfile` in this scope

(COMPILE)